server/command.go

```diff
@@ -120,6 +121,20 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 	}
 }
 
+func (p *Plugin) trackEvent(args *model.CommandArgs, key string, value string) {
+	client, _ := telemetry.NewRudderClient()
+	serverVersion := p.API.GetServerVersion()
+	tracker := telemetry.NewTracker(client, key, serverVersion, manifest.Id, manifest.Version, "mm-plugin-jitsi", true)
```

Comment:
We handle telemetry in a different way. Take a look to this commit:
https://github.com/mattermost/mattermost-plugin-zoom/commit/2f55337ac52b4903a112687a34d69d13be4dc904

Among other things, the problems here are:
- We are creating one client for each event. We usually create one client on startup, or update it on configuration change.
- You are setting this to always track (the final true on this line). This is against the system configuration, that is able to decide whether to send diagnostic information. Check enableDiagnostics on the commit I sent.
- The key in the "NewTracker" function is the secret key. This key is injected during ci. It has nothing to do with the "key value" relationship of the event. See the makefile on the commit I sent.